AgentDiscoveries-Backend/src/main/resources/config.properties

```diff
@@ -1,8 +1,8 @@
 
 # Database properties
 database.url = jdbc:mysql://localhost:3306/agentdiscoveries?useSSL=false
-database.username = CIA
-database.password = Amex1234
+database.username = agentdiscoveries

```

Comment:
We don't really want to check this in, otherwise different developers just keep fighting over these files! Can we remove these from the commit? :)